src/lib/component/util/index.tsx

```diff
@@ -27,6 +28,7 @@ export { default as Timer } from "./Timer";
 export { default as UnauthenticatedRoute } from "./UnauthenticatedRoute";
 export { default as UnmountObserver } from "./UnmountObserver";
 export { default as useBuildInfo } from "./useBuildInfo";
+export { default as useConfigurationProvider } from "./ConfigurationProvider";
```

Comment:
`default` refers to the `ConfigurationProvider` component, you should be able to just use `export { useConfigurationProvider } from "./ConfigurationProvider";`

Reply:
Fixed